CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## [v0.0.20](http://github.com/ndustrialio/contxt-sdk-js/tree/v0.0.21) (2018-07-23)
+
+**Fixed**
+
+* Updated AssetTypes#create to allow for the creation of global asset types
+
+**Changed**
+
+* Now supporting the `data_type` field for `AssetAttributes`
```

Comment:
Is this in another pull request? I don't see any files changed that include `data_type`?

Reply:
I did a bad copy and paste job. Will remove.